package.json

```diff
@@ -24,7 +24,7 @@
 	"icon": "assets/copilot.png",
 	"pricing": "Trial",
 	"engines": {
-		"vscode": "^1.104.0-20250828",
+		"vscode": "^1.104.0-20250905",
```

Comment:
FWIW I don't think changes like this need an engine version and dts version bump (and you should only ever need one of those). It looks like the only functional change is adding an optional statusIcon prop?